scripts/bootstrap.sh.in

```diff
@@ -17,47 +17,47 @@ SALT_MASTER_FILE_CONF=/etc/salt/master.d/99-metalk8s.conf
 SALT_MINION_FILE_CONF=/etc/salt/minion.d/99-metalk8s.conf
 
 die() {
-        echo 1>&2 "$@"
```

Comment:
In general, avoid making changes like this in the scope of a feature PR.

Reply:
Ok I'll remove them and open a dedicated PR.